### PR TITLE
Update to Rails 6.1

### PR DESCRIPTION
**Pull request by truetone**
* Update to Rails 6.1.7.6 and run the app:update task. 446522d0379f9b805908ef262d98d3658a14e1c1
* Update RSpec 861c0e88fb50834ff4f38b24c87660d6f927d549
* Delete .ruby-version file now that this is running in Docker bd9bd5a38c87a36588242a998606df4a5c641b7f
* Run the tests in random order 48b17e672b1bc04755d4d34855684d4c85da27e5
* Stop ignoring CVEs ae1b433a8da4d1dd6a3d7802ecdc516de11a53e2

### CODE REVIEW

**Review by davinlagerroos — APPROVED** [review]
I have one suggestion/question, otherwise this looks good!

**Comment by truetone** [comment]
> I have one suggestion/question, otherwise this looks good!

@davinlagerroos Your suggestion isn't showing up anywhere that I can see.

File: config/environments/production.rb [hunk 1]
```diff
@@ -61,15 +55,19 @@
...
+
   # Use default logging formatter so that PID and timestamp are not suppressed.
   config.log_formatter = ::Logger::Formatter.new
 
   # Use a different logger for distributed setups.
-  # require 'syslog/logger'
+  # require "syslog/logger"
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  config.x.peoplesoft_models_schema = "asr_warehouse_esup"
```
Comment (davinlagerroos) [inline config/environments/production.rb:71]: Not sure how this suggestions got lost.

I think we still need this line to make sure we are talking to the correct schema. I think in prod `asr_warehouse` (the setting from `config/application.rb`) is a bunch of aliases

Comment (truetone) [reply config/environments/production.rb:71]: 🤦 Yes, thanks for catching that.

**Pull request merged**